src/handlers/localstorage/globalSettings.ts

```diff
@@ -64,3 +65,16 @@ export const saveTransactionSignatures = (transactionSignatures: any) =>
  */
 export const getTransactionSignatures = () =>
   getGlobal(TRANSACTION_SIGNATURES, {});
+
+/**
+ * @desc get testnets enabled preference
+ */
+export const getTestnetsEnabled = () => getGlobal(TESTNETS_ENABLED, false);
+
+/**
+ * @desc save testnets enabled preference
+ * @param  {Boolean}  [value]
+ */
+export const saveTestnetsEnabled = (preference: boolean) => {
```

Comment:
also leaving a note here for if this should be in mmkv

Reply:
feel like we should move all settings related stuff to mmkv at the same time if we're going to that